internal/framework/error.go

```diff
@@ -0,0 +1,98 @@
+package framework
+
+import "context"
+
+type ErrorIdentifier string
+
+const WrongRequestDecoding ErrorIdentifier = "WrongRequestDecoding"
+const InvalidRequest ErrorIdentifier = "InvalidRequest"
+const UnprocessableEntity ErrorIdentifier = "UnprocessableEntity"
+const UnknownError ErrorIdentifier = "UnknownError"
+
+type CommonError struct {
```

Comment:
Let's discuss the format of errors. As for me, it is one of the main data types of any framework.

Reply:
https://github.com/go-modulus/demo/pull/2